edgecontext/edgecontext.go

```diff
@@ -17,20 +17,25 @@ import (
 	"github.com/apache/thrift/lib/go/thrift"
 )
 
+// LoIDPrefix is the prefix for all LoIDs.
+const LoIDPrefix = "t2_"
+
 // ErrLoIDWrongPrefix is an error could be returned by New() when passed in LoID
 // does not have the correct prefix.
-var ErrLoIDWrongPrefix = errors.New("edgecontext: loid should have t2_ prefix")
+var ErrLoIDWrongPrefix = errors.New("edgecontext: loid should have " + LoIDPrefix + " prefix")
 
 // ErrNoHeader is an error could be returned by FromThriftContext() when passed
 // in context does not have Edge-Request header set.
 var ErrNoHeader = errors.New("edgecontext: no Edge-Request header found")
 
-// global vars that will be initialized in Init function.
-var (
+// An Impl is an initialized edge context implementation.
+//
+// Please call Init function to initialize it.
+type Impl struct {
```

Comment:
Could this maybe be just the `EdgeContext` struct?

Reply:
We already have [`EdgeRequestContext`](https://pkg.go.dev/github.com/reddit/baseplate.go/edgecontext?tab=doc#EdgeRequestContext), so that name will be more confusing.